packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Deprecations
+
+-   Deprecate bottom margin on `BaseControl`-based components. Set the `__nextHasNoMarginBottom` prop to true to start opting into the new styles, which will become the default in a future version. See linked PR for full list of affected components ([#64408](https://github.com/WordPress/gutenberg/pull/64408)).
```

Comment:
It could be a good idea to list all affected components here. IMHO it's totally fine (and actually useful to our future selves) to add one CHANGELOG line per component. Makes it easier when someone scans the CHANGELOG for suspected changes.

Reply:
Done in 807b1a3